### PR TITLE
Add tiny-compress-images

### PR DESCRIPTION
**Pull request by christianbarkowsky**

### CODE REVIEW

File: meta/christianbarkowsky/contao-tiny-compress-images/de.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+de:
+  title: tiny-compress-images
+  description: >
+    This plugin automatically optimizes your images by integrating with the popular image compression services TinyJPG and TinyPNG.  keywords:
```
[the remark this answers is not in the document]
Comment (christianbarkowsky) [reply meta/christianbarkowsky/contao-tiny-compress-images/de.yml]: Ups ... it was late. I'm so sorry. I will change it tomorrow.

[the remark this answers is not in the document]
Comment (christianbarkowsky) [reply meta/christianbarkowsky/contao-tiny-compress-images/de.yml]: Done.

**Comment by Toflar** [comment]
This PR contains way too many changes. You have to rebase on the current master branch :) Time to learn how to do that :)

**Comment by aschempp** [comment]
I'm closing this due to no activity. Feel free to create a new PR once you have rebased your branch.

**Pull request closed without merging**